icu4c/source/common/dictbe.cpp

```diff
@@ -1381,9 +1432,33 @@ CjkBreakEngine::divideUpDictionaryRange( UText *inText,
     }
     (void)prevCPPos; // suppress compiler warnings about unused variable
 
+    UChar32 lastChar = utext_char32At(inText,rangeEnd);
+    if (!foundBreaks.isEmpty()
+            && foundBreaks.peeki() == rangeEnd
+            && !isOpenPunct(lastChar)
+            && !fNumberWordSet.contains(lastChar)) {
+        foundBreaks.popi();
+        correctedNumBreaks--;
+    }
+
     // inString goes out of scope
     // inputMap goes out of scope
-    return numBreaks;
+    return correctedNumBreaks;
+}
+
+void CjkBreakEngine::loadJpParticleAndAuxVerbs() {
+    UErrorCode error = U_ZERO_ERROR;
+    const char* tags[2] = {"particles", "auxVerbs"};
+
+    ResourceBundle ja(U_ICUDATA_BRKITR, "ja", error);
+    if (U_SUCCESS(error)) {
+       for(int32_t i = 0 ; i < 2 ; i++) {
```

Comment:
add " " between "for" and (" and remove the " " between "0" and "; and "2" and ";".